src/Service/Program.cs

```diff
@@ -23,10 +23,23 @@ public class Program
 
         public static void Main(string[] args)
         {
+            if (!ValidateAspNetCoreUrls())
+            {
+                Console.Error.WriteLine("Invalid ASPNETCORE_URLS format. See: https://github.com/Azure/data-api-builder/issues/2237");
+                Environment.Exit(-1);
+            }
+
             if (!StartEngine(args))
             {
                 Environment.ExitCode = -1;
             }
+
+            static bool ValidateAspNetCoreUrls()
```

Comment:
This should be moved out of `Main` method and a small summary should be added.

Reply:
Why?

Reply:
A local method communicates that its scope is limited to this method. Moving it to the class level implies it might be intended for multiple methods, but it is not. I can move it if "just because we do it that way" is the answer. Otherwise, are you sure?

Reply:
I moved it, but it feels wrong.

Reply:
my reasoning was that i think this could be used by CLI in future, and it usage could be more than just local.